consensus/hotstuff/voteaggregator/vote_aggregator.go

```diff
@@ -360,7 +360,6 @@ func (va *VoteAggregator) tryBuildQC(blockID flow.Identifier) (*flow.QuorumCerti
 	}
 
 	va.createdQC[blockID] = qc
-	va.notifier.OnQcConstructedFromVotes(qc)
```

Comment:
since the notification now moved from vote aggregator to [event handler](https://github.com/onflow/flow-go/pull/1168/files#diff-0cd5bf32d36129e3bf313db65087f712644e35809e4afc35ddd332cec784b723R77), there will be a bit delay from the moment, the qc is created to the moment its processed by event handler. The delay is from the qc go through the event loop.

In order to make both moments clear and be able to measure the delay, I [added an info log when the QC was created by the vote aggregator.](https://github.com/onflow/flow-go/pull/1168/files#diff-9fa312161217502a3ae5a872cbda8043a2afe6d36363e3f47193c0cb32f0800aR220-R225)

Reply:
nice. Thanks.

Reply:
Would be great to have the same log to `CombinedVoteProcessorV3`: https://github.com/onflow/flow-go/blob/d2d9003adf8611a2587c84e83ac3bbbb2aa026aa/consensus/hotstuff/votecollector/combined_vote_processor_v3.go#L231